modify stephs doc PR (#126)
biocore/q2-qemistree: README.md

```

**Note:**
1. The input to `--i-ms2-matches` can be obtained using [Feature-based molecular networking or FBMN](https://gnps.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22FEATURE-BASED-MOLECULAR-NETWORKING%22,%22library_on_server%22:%22d.speclibs;%22%7D) workflow supported in the web-based mass-spectrometry data analysis platform, [GNPS](https://gnps.ucsd.edu/). To use MS2 matches in Qemistree, please download the results of FBMN workflow and import the tsv file in the folder `clusterinfo_summary` as a QIIME2 artifact of type `FeatureData[Molecules]` as follows:

```bash
qiime tools import \
  --input-path /path-to-MS2-spectral-matches.tsv/ \

>   --output-path /path-to-MS2-spectral-matches.qza/ \

Reviewer: ```suggestion
  --input-path path-to-MS2-spectral-matches.tsv \
  --output-path path-to-MS2-spectral-matches.qza \
```

Lastly, Qemistree includes some utility functions that are useful to visualize and explore the molecular hierarchy generated above.

1. Prune molecular hierarchy to keep only the molecules with annotations.

```bash
qiime qemistree prune-hierarchy \
  --i-feature-data classified-merged-feature-data.qza \
  --p-column class \
  --i-tree merged-qemistree.qza \
  --o-pruned-tree merged-qemistree-class.qza
```

Users can choose any of the data columns (`--p-column`) that are in the `classified-merged-feature-data.qza` file to prune the hierarchy. For e.g. '#featureID','kingdom', 'superclass', 'class', 'subclass', 'direct_parent', and 'smiles'. All features with no data in this column will be removed from the phylogeny.

2.1 Generate an annotated qemistree tree in [iTOL](https://itol.embl.de/).

```bash
qiime qemistree plot \
  --i-tree merged-qemistree-class.qza \
  --i-feature-metadata classified-merged-feature-data.qza \
  --p-category class \
  --o-visualization /path-to-qemistree-plot.qzv/
```

> This QZV can be visualized in [iTOL](https://itol.embl.de/) using [Qiime2 Viewer](https://view.qiime2.org); iTOL interface can be used to interact and make visual modifications.

Reviewer: Worthwhile including a note about plots being temporary.

2.2 Add sample metadata to tree tips

> If the user has sample metadata columns by they want to compare samples, Qemistree enables them to visualize feature abundance barcharts at the tips of the tree (abundance or relative abundance) of the feature stratified by the sample metadata column of interest. This can be done as follows:

Reviewer: ```suggestion
If the user has sample metadata columns to compare groups of samples, Qemistree enables them to visualize feature abundance barcharts at the tips of the tree (abundance or relative abundance) of the feature stratified by the sample metadata column of interest. This can be done as follows:

```

2.2.1 Generate the grouped table file using `feature-table group` module in QIIME2:

```bash
qiime feature-table group \
  --i-table feature-table-hashed.qza \
  --p-axis 'sample'

>   --m-metadata-file /path-to-sample-metadata.tsv/ \

Reviewer: ```suggestion
  --m-metadata-file path-to-sample-metadata.tsv \

```

  --m-metadata-column 'disease_vs_healthy' \
  --o-grouped-table /path-to-grouped-feature-table.qza/
```
With the grouped table in hand, the following module can be run to create an annotated iTOL tree.

```bash
qiime qemistree plot \

>   --i-grouped-table /path-to-grouped-feature-table.qza/ \

Reviewer: ```suggestion
  --i-grouped-table path-to-grouped-feature-table.qza \

```
